apps/condo/domains/billing/schema/AllResidentBillingReceiptsService.js

```diff
@@ -97,10 +98,18 @@ const GetAllResidentBillingReceiptsService = new GQLCustomSchema('GetAllResident
                 if (!Array.isArray(serviceConsumers) || !serviceConsumers.length) {
                     return []
                 }
+
                 const processedReceipts = []
                 for (const serviceConsumer of serviceConsumers) {
 
-                    const receiptsQuery = { ...receiptsWhere, ...{ account: { id: serviceConsumer.billingAccount } }, deletedAt: null }
+                    const resident = await getById('Resident', serviceConsumer.resident)
```

Comment:
we also can remove the `serviceConsumer.billingAccount` field or mark it deprecated

Reply:
Yes